### PR TITLE
Bumping to the latest unity and topshelf packages

### PR DESCRIPTION
**Pull request by HEskandari**
This bumps to the latest Unity package (v5.11). Think it needs to be released as major but I haven't changed the nuspec files (although I can).

### CODE REVIEW

File: src/Topshelf.Unity.Sample/Topshelf.Unity.Sample.csproj [hunk 1]
```diff
@@ -1,87 +1,87 @@
...
-    </ProjectReference>
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
-  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
-    <PropertyGroup>
-      <ErrorText>This project references NuGet package(s) that are missing on this computer. Enable NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
-    </PropertyGroup>
-    <Error Condition="!Exists('$(SolutionDir)\.nuget\NuGet.targets')" Text="$([System.String]::Format('$(ErrorText)', '$(SolutionDir)\.nuget\NuGet.targets'))" />
-  </Target>
```
Comment (alexandrnikitin) [inline src/Topshelf.Unity.Sample/Topshelf.Unity.Sample.csproj]: Why are there so many line changes? is it tabs vs spaces?

**Review by alexandrnikitin — COMMENTED** [review]
Thank you for the PR!

I see many lines affected. I assume it's because you use tabs?
Also I don't see any code changes related to the new Abstractions version. Won't it be enough to change [the nuget dependency range](https://github.com/alexandrnikitin/Topshelf.Unity/blob/d620fb442028fa99a113a273fd4ee7e61980b104/src/Topshelf.Unity/Topshelf.Unity.nuspec#L14)

File: src/Topshelf.Unity.Sample/Topshelf.Unity.Sample.csproj (hunk 1 above)
Comment (HEskandari) [reply src/Topshelf.Unity.Sample/Topshelf.Unity.Sample.csproj]: Could be. Would be good to drive that convention using a `.editorconfig. file. I'll update the PR.

**Comment by HEskandari** [comment]
@alexandrnikitin I think changing the dependency range should suffice. I'll send you an update shortly.

**Comment by HEskandari** [comment]
@alexandrnikitin Fixed the whitespace issues. Also updated the nuspec file.

**Review by alexandrnikitin — APPROVED** [review]

**Comment by alexandrnikitin** [comment]
Great! Merged.
Thank you!

**Comment by alexandrnikitin** [comment]
Pushed [v4.0.0 to NuGet](https://www.nuget.org/packages/Topshelf.Unity/4.0.0)

**Pull request merged**